stdlib/2and3/ssl.pyi

```diff
@@ -141,21 +141,38 @@ if sys.version_info >= (3, 6):
         OP_SINGLE_ECDH_USE: int
         OP_NO_COMPRESSION: int
         OP_NO_TICKET: int
-        OP_ENABLE_MIDDLEBOX_COMPAT: int
         if sys.version_info >= (3, 7):
             OP_NO_RENEGOTIATION: int
-else:
-    OP_ALL: int
-    OP_NO_SSLv2: int
-    OP_NO_SSLv3: int
-    OP_NO_TLSv1: int
-    OP_NO_TLSv1_1: int
-    OP_NO_TLSv1_2: int
-    OP_CIPHER_SERVER_PREFERENCE: int
-    OP_SINGLE_DH_USE: int
-    OP_SINGLE_ECDH_USE: int
-    OP_NO_COMPRESSION: int
+            OP_NO_TLSv1_3: int
+        if sys.version_info >= (3, 8):
+            OP_ENABLE_MIDDLEBOX_COMPAT: int
+
+OP_ALL: int
+OP_NO_SSLv2: int
+OP_NO_SSLv3: int
+OP_NO_TLSv1: int
+OP_NO_TLSv1_1: int
+OP_NO_TLSv1_2: int
+OP_CIPHER_SERVER_PREFERENCE: int
+OP_SINGLE_DH_USE: int
+OP_SINGLE_ECDH_USE: int
+OP_NO_COMPRESSION: int
```

Comment:
They're actually instances of the Options enum on 3.6+.

Reply:
oh I see!! Whoops! Fixing